Earthfile

```diff
@@ -1,5 +1,8 @@
 VERSION 0.6
-ARG BASE_IMAGE=quay.io/kairos/core-ubuntu:latest
+
+# TODO: Find a way to automatically bump this
+ARG BASE_IMAGE=quay.io/kairos/ubuntu:23.10-core-amd64-generic-v2.5.0
```

Comment:
I think you should be able to do something like:

https://github.com/kairos-io/kairos/blob/master/Earthfile#L11

and 

https://github.com/kairos-io/kairos/blob/master/renovate.json#L24

but you will need to extend the flavor_release part.

Maybe we should talk about introducing some other tags, e.g. this could be something like `ubuntu:rolling-core-amd64-generic-stable` or so

Reply:
Nice! I add the renovate things. Should be ok for now.